internal/health/status/check.go

```diff
@@ -36,7 +36,8 @@ var (
 	ConditionChecks = []ConditionCheckFn{
 		condition.AllMembersReadyCheck,
 		condition.ReadyCheck,
-		condition.BackupReadyCheck,
+		condition.FullSnapshotBackupReadyCheck,
```

Comment:
you have removed `BackupReadyCheck` here but you have also changed and kept the implementation of `BackupReady` in `check_backup_ready.go`.  Any reason for that?